overlay/overlay_launchers.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2005-2016 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_OVERLAY_LAUNCHERS_H_
+#define MUMBLE_OVERLAY_LAUNCHERS_H_
+
+static const char *overlayLaunchers[] = {
+	"Steam.exe", // Steam
+	"EALaunchHelper.exe", // Origin
+	"Battle.net.exe", // Battle.net
+	"GalaxyClient.exe", // GOG Galaxy
+	"UbisoftGameLauncher.exe", // Uplay
+	"UbisoftGameLauncher64.exe", // Uplay
+
```

Comment:
Can you check if BethesdaNetLauncher is the immediate parent of games launched through their launcher?